File: api_wrapper/thecatapi/python/src/models/category.py
import typing

import model


class Category(model.Model):

    def __init__(self,
                 id: typing.Optional[int] = None,
                 name: typing.Optional[str] = None,
                 **other_fields) -> None:
        super().__init__(id, **other_fields)
        self.name = name

File: api_wrapper/thecatapi/java/src/main/java/model/Category.java
package model;

import java.util.Map;

public class Category extends Model {
    private final String name;

    public Category(String id, Map<String, Object> data) {
        super(id, data);
        this.name = String.valueOf(data.get("name"));
    }

    public String getName() {
        return name;
    }
}
